specs/features/step_definitions/citizenship.js

```diff
@@ -119,7 +119,7 @@ const completeCitizenshipForeignPassports = (promise) => {
 }
 
 const navigateToSection = (section) => {
-  const selector = '.section a[title="' + section + '"]'
+  const selector = '.usa-sidenav-list a[aria-controls="/form/' + section + '"]'
```

Comment:
@afeld when I use `section` here is places "Citizenship" and not "citizenship" lowercase. I'm unsure why? Above section is set to lower case on line 28. Any ideas why? 

I'm getting this error ```ERROR: Unable to locate element: ".usa-sidenav-list a[aria-controls="/form/Citizenship"]" using: css selector```

Reply:
Did you look where `navigateToSection` is called from?